frontend/components/Card/CardAbout.vue

```diff
@@ -1,11 +1,38 @@
 <template>
   <div
-    class="flex flex-col items-center w-full h-min card-style sm:items-start"
+    class="relative flex flex-col items-center w-full h-min card-style sm:items-start"
   >
-    <div class="px-1 py-1 sm:px-2 sm:py-2">
-      <p>About</p>
-      <p>{{ description }}</p>
-      <p>This about card describes an {{ aboutType }}</p>
+    <div class="bg-slate-400 w-10 h-10 sm:w-16 sm:h-16 absolute right-0 m-3 rounded-md"><img src="" alt=""></div>
+    <div class="px-4 py-3 sm:px-2 sm:py-2 w-full">
+      <div class="flex justify-start items-center gap-3 mb-3">
+        <h3 class="text-left responsive-h3 font-display">About</h3>
+        <Icon name="bi:pencil-square" size="1.2em" />
+      </div>
+      <button
+        class="px-2 py-1 border-2 border-light-cta-orange bg-[#F1993D4D] text-light-cta-orange rounded-2xl flex items-center gap-2 mb-3"
+      >
+        <Icon name="bi:globe" size="1.2em" />
+        Housing and Community
+      </button>
```

Comment:
I suggest using the TopicMarker component instead, as it is already created and styled and used in other places.

Reply:
oh cool thanks, will make the changes, still working on this for #129

Reply:
Thank you, @Mradosi!! :)

Reply:
@andrewtavis @Mradosi  Added requested changes above and some more changes for #129

Reply:
Totally fine with updating Tailwind, @momanyisamuel :) Do you want to send that change along as well?

Reply:
done 👍